custom/logistics/api.py

```diff
@@ -250,30 +261,38 @@ def sms_user_sync(self, ilsgateway_smsuser, username_part=None, password=None,
                 user.save()
         return user
 
-    def add_language_to_user(self, logistics_sms_user, domains=None):
-        if not domains:
-            domains = []
+    def _reassign_number(self, user, phone_number):
+        v = VerifiedNumber.by_phone(phone_number, include_pending=True)
+        if v.domain in self._get_logistics_domains():
+            v.delete()
+            user.save_verified_number(self.domain, phone_number, True)
```

Comment:
@gcapalbo This line still raises PhoneNumberInUseException. I wonder how it's possible. I'm pretty sure that verified number is deleted because if it fails on some particular user and then I rerun task It works fine for that particular user. Maybe It's better reassign verified number by changing domain and owner_id instead of deleting it, what do you think?